src/components/BreedsList.vue

```diff
@@ -98,34 +99,36 @@ export default {
       isScrollToTopActive: false,
       lastBreedState: [],
       indexClicked: 0,
+      store: useStore(),
```

Comment:
morem priznat da še nism vidu tega načina dodajanja stora v komponento.. jaz sem vedno delal preko mapperjev.. kot sta `mapActions` in `mapState`.

Ubistvu ti v komponenti rabiš actions in pa getters ane? actions za manipuliranje date in getters za "pridobit" to dato v komponento.

tkole dodaš getters: https://pinia.vuejs.org/core-concepts/getters.html#Without-setup-
in pa actions: https://pinia.vuejs.org/core-concepts/actions.html#Without-setup-

in pol jih imaš na `this` in jih lahko uporabljaš, s tem tud pridobiš reactivity getterjev, ne vem če ga imaš v tvojem trenutnem primeru

Reply:
`setup()` je dodana v vue 3, samo da boš vedu pri branju dokumentacije.. vue 2, ki jo ti uporabljaš uporablja options API

Reply:
Hvala za obrazložitev, vrjetno, če mi nebi povedal da se to uporablja, bi najbrž čist narobe uporabljov vse skupaj ;)